src/redirect.cc

```diff
@@ -200,7 +200,7 @@ redirectStats(StoreEntry * sentry)
         return;
     }
 
-    redirectors->packStatsInto(sentry, "Redirector Statistics");
+    redirectors->packStatsInto(sentry->packer(), "Redirector Statistics");
```

Comment:
So, for example, this should look something like this:

```
StoreEntryPacker packer(*sentry);
redirectors->packStatsInto(&packer, "Redirector Statistics");
```

Reply:
Can we do differently: create and return a static StoreEntryPacker within StoreEntry::packer()?
E.g.:

```
Packable *StoreEntry::packer() {
    static StoreEntryPacker packer;
    packer.setEntry(*this);
    return &packer;
}
```

This looks safe if no packers are created within any of Packable calls.

Reply:
Done in 67b47e8.